src/com/carolinarollergirls/scoreboard/defaults/DefaultScoreBoardModel.java

```diff
@@ -617,11 +690,25 @@ public void scoreBoardChange(ScoreBoardEvent event) {
 	};
 	protected ScoreBoardListener lineupClockListener = new ScoreBoardListener() {
 		public void scoreBoardChange(ScoreBoardEvent event) {
-			if (settings.getBoolean("ScoreBoard." + Clock.ID_LINEUP + ".AutoStart")) {
+			if (settings.getBoolean(SETTING_AUTO_START)) {
 				_possiblyAutostart();
 			}
 		}
 	};
+	protected ScoreBoardListener timeoutClockListener = new ScoreBoardListener() {
+		public void scoreBoardChange(ScoreBoardEvent event) {
+			if (settings.getBoolean(SETTING_AUTO_END_TTO) &&
+					!getTimeoutOwner().equals(TIMEOUT_OWNER_NONE) &&
```

Comment:
Why not explicitly check if it's team 1 or team 2?

Reply:
This way it's all &&, checking for teams would add an ||.